lib/index.js

```diff
@@ -174,15 +174,16 @@ function walk (opts, nodes) {
       return m
     }
 
+    const ignoredBefore = delimitersSettings[1].text[0]
+    const ignoredAfter = delimitersSettings[1].text[1]
+    const toReplace = new RegExp(`@${ignoredBefore}`, 'g')
+    let ignoredExpression = new RegExp(`@${ignoredBefore}(.+?)${ignoredAfter}`, 'g')
```

Comment:
I think it’s better to put on line 154

`ignoredExpression` I think it's worth renaming to `ignoredNodeExpression`